### ISSUE TITLE
Найти референсы дизайна

### ISSUE THREAD

**Issue by rozavetrov** [kaverhovsky/todo-list-app#2]
Основной референс - [Microsoft To Do](https://to-do.live.com/)

![Image](https://github.com/kaverhovsky/todo-list-app/assets/40857989/b1258ed2-661e-46ff-875b-2ed149a2681a)

**Comment by kaverhovsky** [comment]
Не очень понятен референс. Поподробнее, пожалуйста

**Comment by rozavetrov** [comment]
> Не очень понятен референс. Поподробнее, пожалуйста

Согласен. Референс будет детально оформлен в [Google Slides](https://www.google.com/slides/about/)
